### ISSUE TITLE
Show full commands running

### ISSUE THREAD

**Issue by neighthan** [wookayin/gpustat#50]
Would you support an option to show the full commands being run instead of just, e.g, `python`? These could be rather long, so one option might be to put each of them on a new line below the GPU that they're on (maybe indented a bit). Since `gpustat` seems to focus on being very concise, I'd understand if not. I have my own modification of `nvidia-smi` to show more about the processes running on each GPU, and I was planning to make it more concise when I ran across your project. I just want to check if you'd support this so I don't have to duplicate some of your work here `nividia-smi` if so.

**Comment by wookayin** [comment]
I also feel the necessity to display more comprehensive process information, under a specific command-line flag/options. It would not hurt our focus on being concise that much.

I am considering something like this (everything is just sketchy):

```
gpustat --show-process-info
[0] TITAN Xp         | 28'C,  12 % |   903 / 12196 MB | python/25647(893M) python/12345(0M)
  ├─ 25647 (100%): python train.py --alpha 1.0 --beta 2.0
  └─ 12345 (50 %): python test.py
[1] TITAN Xp         | 28'C,  12 % |   903 / 12196 MB | 
```

It might be a little bit verbose. The flag/option name is something we need to think about more and decide. Any suggestion or thoughts?

**Comment by neighthan** [comment]
The format there looks right to me; the ├─ on the left makes it even better than what I was picturing. What is the percentage shown after the PIDs? `--show-process-info` sounds fine to me or `-f`/`--full-process-info` (since `-p`/`--show-pid` is also showing some process info). Maybe starting with `show` is better for consistency, though? Maybe `--show-full-cmd` to be more closely related to `--show-cmd`.

**Comment by bethune-bryant** [comment]
I have a PR open that adds this feature. Feel free to provide any feedback. @neighthan

**Comment by KindDragon** [comment]
Please release a new version with this feature

**Comment by wookayin** [comment]
This was implemented in #65. Will make a release soon, but in the meantime please try installing `gpustat==1.0.0b1`.
